Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/ResourceDownloadScriptHandler.swift

```diff
@@ -59,9 +75,7 @@ class ResourceDownloadManager: TabContentScript {
   }
 
   static func downloadResource(for tab: Tab, url: URL) {
-    let token = UserScriptManager.securityTokenString
-
-    tab.webView?.evaluateSafeJavaScript(functionName: "D\(token).download", args: [url.absoluteString], contentWorld: .defaultClient) { _, error in
+    tab.webView?.evaluateSafeJavaScript(functionName: "window.__firefox__.\(downloadName).download", args: [url.absoluteString], contentWorld: self.scriptSandbox) { _, error in
```

Comment:
Isn't the downloadName user controllable?

Reply:
Fixed. This script is sandboxed so no need for me to randomize the handler.